tests/golden/copy-command/copy-command.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "copy an entry" {
+  run coffer create /a
+  assert_output "[SUCCESS] Entry created at '/a'."
+
+  run coffer copy /a /b
+  assert_output "[SUCCESS] Copied '/a' to '/b'."
+
+  run cleanOutput coffer view /
+  cat copy-an-entry.gold | assert_output -
+}
+
+@test "copy a directory" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer copy /a /
+  assert_output "[SUCCESS] Copied '/a/b/c' to '/b/c'."
+
+  run cleanOutput coffer view /
+  cat copy-a-directory.gold | assert_output -
+}
+
+@test "copy one directory to other" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer create /c/d/e
+  assert_output "[SUCCESS] Entry created at '/c/d/e'."
+
+  run coffer copy /a /c
+  assert_output "[SUCCESS] Copied '/a/b/c' to '/c/b/c'."
+
+  run cleanOutput coffer view /
+  cat copy-one-directory-to-other.gold | assert_output -
+}
+
+@test "trying to copy an entry to directory" {
```

Comment:
It's hard to tell what the purpose of these tests is by looking at their names alone.

Test names should be descriptive. May I suggest:
* "trying to copy an entry to directory" -> "copying an entry fails when it clashes with an existing directory"
* "copy a directory without force" -> "copying a directory fails when any entry clashes with an existing entry"
* "copy a directory with force" -> "copying a directory succeeds when any entry clashes with an existing entry and --force is used"

Reply:
Think we should also add a test for "copying a directory fails when any entry clashes with an existing directory"

Reply:
I'll add this test in #14, because now it fails/

Reply:
It should work here though:

```
$ coffer create /a/b  
$ coffer create /x/b/c
$ coffer copy /a /x   
[ERROR] The following entries cannot be copied because a directory already exists at the destination.

Cannot copy '/a/b' to '/x/b'.
```

The entry `/a/b` clashed with the existing directory `/x/b`.

----

#14 aims to solve a different problem: when copying an entry, and we need to create a directory for that entry to live in, but that directory path is already taken by another entry.

E.g., if we copy an entry to `/a/b/c`, we need to create the directories `/a` and `/a/b`. But if the paths `/a` or `/a/b` are already taken by an existing entry, the operation fails.

Reply:
Done